index.html

```diff
@@ -755,6 +744,14 @@ <h2>
             </li>
           </ul>
         </dd>
+        <dt>
+          <dfn>Patronizing</dfn>
+        </dt>
+        <dd>
+          <p>
+            Language or behaviour that may appear kind or helpful but conveys a feeling of superiority or condescension.
```

Comment:
1. I don't think changing "betrays" to "conveys" is an improvement.  Dictionary definitions are crafted extremely carefully.  "Betrays" implies that the person patronizing another is _pretending_ to be kind or helpful, while the actual motive is not so kind.  This is an important element in the meaning of the word.  If we include this definition I think "betrays" should be retained.

2. If we include this definition, we should give credit to its source.

3. I would not object if others think a definition of "patronizing" is important to include, but my preference would be to not include one, in order to keep the document from growing larger.  In general, I do not think the glossary should be defining terms whose meaning is readily found in any dictionary.  It should only be used for terms that have special meaning in this document.